### ISSUE TITLE
Cannot force delete more than 10 flows

### ISSUE THREAD

**Issue by orionarcher** [Matgenix/jobflow-remote#169]
I am unable to delete more than 10 flows at a time, even with explicit confirmation and setting the `--force` flag. Is this the intended behavior or a bug?

<img width="845" alt="image" src="https://github.com/user-attachments/assets/78171f84-9c82-4de1-8759-a65a065a4e9c">

<img width="855" alt="image" src="https://github.com/user-attachments/assets/cc3caf88-16eb-4daf-9ca5-790a9c152a63">

**Comment by davidwaroquiers** [comment]
Thanks @orionarcher for raising that issue. It indeed seems to be a bug. Should be very easy to fix.

@gpetretto I think it is this line here: https://github.com/Matgenix/jobflow-remote/blob/develop/src/jobflow_remote/cli/flow.py#L239 where confirmed should be passed and set to True.

I'm just not completely sure about how to deal with the less than 10 flows and more than 10 flows. Should we tell the user to use the --force/-f option if there are more than 10 flows, in addition to the confirmation ? Or add another safeguard like "enter today's date" as, if the user does that by error, it can really be problematic ? @orionarcher maybe you can also tell what you'd prefer from the above and what you think about this ?

@orionarcher Note that you could do this programmatically with the python API yourself in the meantime with something like:

```
from jobflow_remote import JobController
from jobflow_remote.jobs.state import FlowState
jc = JobController.from_project_name("yourprojectname")

flows_info = jc.get_flows_info(states=FlowState.RUNNING)
to_delete = [fi.flow_id for fi in flow_info]
jc.delete_flows(flow_ids=to_delete, delete_output=True, confirm=True)
```

**Comment by gpetretto** [comment]
Yes, that is the line. Actually I have already fixed it here: https://github.com/Matgenix/jobflow-remote/pull/154 by just adding `confirm=True` there. I think this was more of a safeguard in case you call the function directly in the `JobController`. For the CLI this will be applied without asking anythin if passing the `--force` option, otherwise the the code will tell the user `"[red]This operation will [bold]delete {len(flows_info)} Flow(s)[/bold]. Proceed anyway?[/red]"`. 
Do you think this will need a stronger check, if the number of flows is large?

Alternatively it couldbe made similar to `reset`. Where the maximum number of Flows that can be deleted is a parameter. So if the default value is exceed the user will need to explicitly increase the maximum value.

**Comment by davidwaroquiers** [comment]
I think the max number parameter is a good idea, makes it also similar to reset.

**Comment by orionarcher** [comment]
Thank you for the quick response. I think either interface option would work well.

**Comment by gpetretto** [comment]
This should be fixed in #154.
Thanks for reporting this.

**Comment by orionarcher** [comment]
I pulled 154 and it's working for me now, thank you!
